infrastructure/vars.tf

```diff
@@ -937,6 +937,11 @@ variable "backend_api_requests_log_group_name" {
   default     = "cyhy-staging-backend-api-requests"
 }
 
+variable "worker_ecs_task_execution_role_arn" {
```

Comment:
Same here. I don't think we need unless I missed it. If so, we can reference the actual terraform resource